trending_data/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file suppose to be commited? It seems unrelated to this change? Same for the .settings/ file below.

Reply:
This has been eliminated as the file is not used